.ibm/images/Dockerfile

```
@@ -16,6 +16,12 @@ ENV CI=1 \
     GOPATH="/go" \
     PATH="$GOPATH/bin:/usr/local/go/bin:$PATH"
 
+# Copy the .yarnrc.yml file to the environment
+COPY .yarnrc.yml /root/.yarnrc.yml
+
+# Extract the correct version from the file and install
+RUN YARN_VERSION=$(awk '/yarnPath:/ {print $2}' /root/.yarnrc.yml | awk -F'[-.]' '{print $(NF-1)"."$NF}') && \
```

Is the .yarnrc.yml file being used by the utils.sh file? if not we can delete it here once you've scraped it.

```suggestion
RUN YARN_VERSION=$(awk '/yarnPath:/ {print $2}' /root/.yarnrc.yml | awk -F'[-.]' '{print $(NF-1)"."$NF}') && rm -f /root/.yarnrc.yml
```

Or else you can keep it like this:

```suggestion
RUN YARN_VERSION=$(awk '/yarnPath:/ {print $2}' /root/.yarnrc.yml | awk -F'[-.]' '{print $(NF-1)"."$NF}')
```

---

@nickboldt  I'm using `YARN_VERSION` here ` npm install -g typescript yarn@$YARN_VERSION && \`